spark/v4.0/build.gradle

```diff
@@ -33,6 +34,7 @@ configure(sparkProjects) {
         force "com.fasterxml.jackson.module:jackson-module-scala_${scalaVersion}:${libs.versions.jackson215.get()}"
         force "com.fasterxml.jackson.core:jackson-databind:${libs.versions.jackson215.get()}"
         force "com.fasterxml.jackson.core:jackson-core:${libs.versions.jackson215.get()}"
+        force "org.antlr:antlr4:${antlrVersion}"
```

Comment:
Why do we have to force here? Shouldn't we be ok just pulling the Antlr out of the spark version?

Reply:
Shouldn't need `force`. Fixed. Thanks